assets/src/js/query_loop_carousel.js

```diff
@@ -1,8 +1,16 @@
 import {v4 as uuid} from 'uuid';
 
+const ARROW_DIRECTIONS = ['prev', 'next'];
+
+const removeArrows = layout => ARROW_DIRECTIONS.forEach(direction => {
+  const controlBtn = layout.querySelector(`.wp-block-buttons .carousel-control-${direction}`);
+  controlBtn?.parentNode.removeChild(controlBtn);
```

Comment:
Should we check whether the `parentNode` also exists?

```suggestion
if (controlBtn && controlBtn.parentNode) {
  controlBtn.parentNode.removeChild(controlBtn);
}
```